lib/bats-core/tracing.bash

```diff
@@ -0,0 +1,152 @@
+bats_capture_stack_trace() {
```

Comment:
indentation broken in this file

Reply:
Yeah, my editor had a mind of its own and I noticed too late. I'll look into shfmt as you advised above.